webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -248,6 +249,7 @@ class TechniqueWriter (
       libUpdate  <- techLibUpdate.update(modId, committer, Some(s"Update Technique library after creating files for ncf Technique ${technique.name}")).
                       toIO.chainError(s"An error occured during technique update after files were created for ncf Technique ${technique.name}")
      } yield {
+      Seq("/opt/rudder/share/rudder-lang/tools/tester.sh", technique.name).run
```

Comment:
You should not run a command with sys.process. 

It can throw exception and it will break the beahvior of Rudder here

You want to have something like this 

```
RunNuCommand.run(Cmd("/opt/rudder/share/rudder-lang/tools/tester.sh", technique.name :: Nil, Map())).either match {
  case Left(error: Error) => ApplicationLogger.error(error.getMessage)
  case Right(_)  => ApplicationLogger.debug(s"rudder lang tester called for technique ${technique.bundleName.value}")
}

Reply:
On a side note, don't you want the technique id/bundle_name instead ? here you are using the technique human name which is not the technique identifier!!

Reply:
I'll look into it, thank you
More fixes are coming along

Well, I need the `technique.name` because it allows me to get the technique path of the created file. 
But maybe I could use the technique id for log purposes